### PR TITLE
Contributing file add

### PR DESCRIPTION
**Pull request by kunalvishnoi**
as per #2 contributing file add

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ 
+ Please note we have a code of conduct, please follow it in all your interactions with the project.
+ 
+ 
+ 
+ 
+ ## Getting Started
+ 
+ * [Fork the repo!](https://github.com/ncs-jss/ncs-jss.github.io/fork) and clone it.
+ * Navigate to the project: `cd ncs-jss`.
```
Comment (ankitjain28may) [inline Contributing.md]: It should be `cd ncs-jss.github.io`

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ * If you've started working on an issue, leave a comment as it helps avoid duplicate work.
+ * Make your changes.
+  Run "git commit" and add your message describing what you have changed. If you have made changes that address an issue include the text "Closes #1" (where 1 would  be   the issue number) to your commit message.
+  
+ * Submit a [pull request](https://github.com/ncs-jss/ncs-jss.github.io/pulls). Once you have committed your changes locally and pushed them to your fork of the repo, a banner with a **Compare & pull request** button should appear on the main repo page to submit a pull request.
+ 
+## Code of Conduct
+ 
+
+##Our Pledge
```
Comment (ankitjain28may) [inline Contributing.md]: Give a single space between `###` and heading

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+##Our Standards
```
Comment (ankitjain28may) [inline Contributing.md]: Single space and check the same for all the follow up headings

**Review by ankitjain28may — CHANGES_REQUESTED** [review]

**Comment by kunalvishnoi** [comment]
@ankitjain28may  as requested changes has been done.

**Pull request merged**